enos/modules/vault_cluster/scripts/install-packages.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
```

Comment:
As they do the same thing, we probably ought to remove this script and use the `install_packages` module in `vault_cluster` instead.